### PR TITLE
fixed flapper by reducing the number of samples upserted

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

File: tests/jobQueue/v1/getBulkUpsertStatus.js [hunk 1]
```diff
@@ -171,13 +173,13 @@ describe('tests/jobQueue/v1/getBulkUpsertStatus.js, ' +
     });
   });
 
-  it('Even when hundreds of upserts fail, the valid upsert should ' +
+  it('Even when lots of upserts fail, the valid upsert should ' +
```
Comment (iamigo) [inline tests/jobQueue/v1/getBulkUpsertStatus.js]: The code under test here is the same whether it's hundreds or 20 or 3, right?

Comment (shriramshankar) [inline tests/jobQueue/v1/getBulkUpsertStatus.js]: yes is it the same for most part. Only change is in the number of samples upserted.

Comment (iamigo) [inline tests/jobQueue/v1/getBulkUpsertStatus.js]: smaller number of samples could mean smaller setTimeout?
(500ms times multiple tests is expensive)

Comment (shriramshankar) [inline tests/jobQueue/v1/getBulkUpsertStatus.js]: Yes, definitely. But not sure where to draw the line. 400ms?

Comment (iamigo) [inline tests/jobQueue/v1/getBulkUpsertStatus.js]: Yeah, I guess the whole point is we can't afford to have it be flappy, so I guess it's more important to err on the side of slower but solid.

**Pull request merged**